jax/experimental/jax2tf/jax2tf.py

```diff
@@ -56,11 +56,17 @@ def _is_tfval(v: TfVal) -> bool:
   if isinstance(v, (tf.Tensor, tf.Variable)):
     return True
   try:
-    tf.convert_to_tensor(v)
+    safe_convert_to_tensor(v)
     return True
   except ValueError:
     return False
 
+def safe_convert_to_tensor(v):
```

Comment:
I am not really sure whether we should increase the API surface, but if we do we should add a docstring. This function should be used rarely, only when you want to pass jnp.bfloat16 arrays to jax2tf code (very rare, because most often that code will be executed in a TF world where jnp.bfloat16 does not exist)

Reply:
See my comment for the tests, I think we can indeed remove safe_convert_to_tensor and move its body to the test.

Reply:
Addressed and move to both the test and the conversion in `tf_test_util`.